packages/js/product-editor/src/components/details-name-block/edit.tsx

```diff
@@ -76,10 +76,13 @@ export function Edit() {
 		}
 	);
 
-	const nameIsValid = useValidation(
-		'product/name',
-		() => Boolean( name ) && name !== AUTO_DRAFT_NAME
-	);
+	const nameIsValid = Boolean( name ) && name !== AUTO_DRAFT_NAME;
```

Comment:
I think we should keep the validation hook. The validation hook prevent the editor from been saved. Also if it's removed then will be hard to identify/fix the logic once the validation updates will be implemented.

Reply:
Good call 👍🏻 . Done in https://github.com/woocommerce/woocommerce/pull/37623/commits/56a187ad22483b602ff238c869e6982abcb4afaf